deferred-leveldown.js

```diff
@@ -3,8 +3,15 @@
 const AbstractLevelDOWN = require('abstract-leveldown').AbstractLevelDOWN
 const inherits = require('inherits')
 const DeferredIterator = require('./deferred-iterator')
-const deferrables = 'put get del batch clear'.split(' ')
-const optionalDeferrables = 'approximateSize compactRange'.split(' ')
+const DeferredChainedBatch = require('./deferred-chained-batch')
+const getCallback = require('./util').getCallback
+
+const deferrables = ['put', 'get', 'getMany', 'del', 'batch', 'clear']
+const optionalDeferrables = ['approximateSize', 'compactRange']
+
+const kNut = Symbol('nut')
```

Comment:
what does nut stand for?

Reply:
deferred-leveldown is a shell (db) around a nut (another db). I wanted something more explicit than `db._db.db`

Reply:
Right, `db._db.db` isn't great.  What about `wrappedDb` or `innerDb` instead? Something that is less likely to need explanation

Reply:
I hoped this would be enough to explain it:

https://github.com/Level/deferred-leveldown/blob/c197496adf54b7b44bb74838dcc342f377135ece/deferred-leveldown.js#L27

If we start using the term "nut" in more places (I picked it up from dominic tarr's sublevel stuff, although there it was more obvious because it was paired with variables called "shell"), it just needs explanation once and we have a nice short word.

Reply:
I don't care as much about brevity, and a nut without a shell doesn't make it obvious what the nut means.

Reply:
Fair enough. Will change

Reply:
It's less exciting this way 😅 But I think will cause less issues